tests/storage/configuration.go

```diff
@@ -228,10 +228,12 @@ var _ = Describe("[sig-storage] Storage configuration", decorators.SigStorage, d
 			By("setting the disk to use custom block sizes")
 			logicalSize := uint(16384)
 			physicalSize := uint(16384)
+			discardGranularity := uint(16384)
```

Comment:
It's probably fine to piggyback on an existing func test but ultimately we are much better off with all of these as unit tests